portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSingleSelectionField.testcase

```diff
@@ -73,13 +73,65 @@ definition {
 	}
 
 	@description = "Verify that a Single Selection Field can be required"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanBeRequired {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-132192 CanBeRequired pending implementation
+		ProductMenu.gotoPortlet(
+                        category = "Content &amp; Data",
+                        portlet = "Forms");
+
+                FormsAdminNavigator.gotoAddForm();
+
+                Form.editName(formName = "Form 1");
+
+                Form.gotoAddField(
+                        fieldPositionNumber = "0",
+                        fieldType = "Single Selection",
+                        formPageNumber = "0");
+
+		var fieldSingleSelectionField = FormFields.getFieldName();
+
+                FormFields.enableSwitcher(fieldName = "Required Field");
+
+                Form.save();
+
+                Form.publishForm();
+
+                FormsAdminNavigator.gotoPublishCopyURLLink();
+
+                var publishedFormPublicURL = Form.getPublishedFormPublicURL(
+                        publishedFormPublicURL = "${publishedFormPublicURL}");
+
+                Navigator.openSpecificURL(url = "${publishedFormPublicURL}");
+
+		FormViewBuilder.validateFieldLabelRequired(fieldLabel = "Single Selection");
+
+                Click(
+                        key_text = "Submit",
+                        locator1 = "Button#ANY");
+
+                FormFields.viewFieldValidationErrorMessage(
+                        fieldName = "SingleSelection",
+                        fieldValidationErrorMessage = "This field is required.");
+
+		FormFields.viewRadioLabel(
+			fieldLabel = "Single Selection",
+			fieldName = "${fieldSingleSelectionField}");
+
+		FormFields.selectRadioOption(
+			fieldName = "${fieldSingleSelectionField}",
+			radioOption = "Option");
+
+                Click(
+                        key_text = "Submit",
+                        locator1 = "Button#ANY");
+
+                AssertElementPresent(locator1 = "Message#SUCCESS");
```

Comment:
You can just use `FormPortlet.submitSuccessfully()`, which is going to submit the form and assert the success message has been displayed.